tests/unit_tests/test_wave_06.py

```diff
@@ -109,8 +110,16 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert result
```

Comment:
You could do `assert result is True` to make sure that an actual `True` comes out of the function instead of just a truthy value